ciao-4.11/contrib/lib/python3.5/site-packages/sherpa_contrib/chart.py

```diff
@@ -205,6 +201,16 @@ def save_chart_spectrum(outfile, clobber=True, verbose=True,
         Should a message be displayed indicating that the file
         has been created? This is only used when outfile is not
         ``None``.
+    id : int or string or None
+        If id is None then the default Sherpa id is used. This
+        dataset must have a grid and source model defined.
+    elow, ehigh, ewidth : number or None
+        Only used if all three are given, otherwise the grid
+        of the data set is used. The elo value is the start
+        of the low-energy bin, ehigh is the end of the high-energy
```

Comment:
low-energybin -> bin of lowest energy and the same for high-energy-bin. When we talk about Chandra we say things like "The low-energy signal", whereas here "low-energy" cna actually be a high energy, e.h. `elow=9, ehigh=10, ewdith=0.1`, so I suggest rewording to avoid confusion.

Reply:
I'm changing this to the following (in `chart.py` and `marx.py`). I am not 100% happy with using left/right edge, but I also don't think it is worth obsessing over too much.

```
    elow, ehigh, ewidth : number or None
        Only used if all three are given, otherwise the grid
        of the data set is used. The elo value gives the start
        of the grid (the left edge of the first bin) and ehi
        is the end of the grid (the right edge of the last bin),
        and ewidth the bin width, all in keV.
```

Perhaps I should centralize this documentation rather than copy-pasting, but that has a user cost (having to look it up), so I am not going to do that now.